assests/styling/customer.scss

```diff
@@ -0,0 +1,112 @@
+.customer__table {
```

Comment:
Als je de tabel kleiner wilt hebben kan je in de html in plaats van '<div class="container-fluid">' ook '<div class="container">'. Dit is ook een bootstrap class die zorgt dat je pagina nooit te breed is en mooi kleiner wordt richting mobile.